MessagingService.Testing/TestData.cs

```diff
@@ -38,6 +40,80 @@ public class TestData
 
         public static String ProviderEmailReference = "ProviderEmailReference";
 
+        public static MessageStatus MessageStatusDelivered = MessageStatus.Delivered;
+
+        public static MessageStatus MessageStatusRejected = MessageStatus.Rejected;
+        
+        public static MessageStatus MessageStatusFailed = MessageStatus.Failed;
+        
+        public static MessageStatus MessageStatusBounced = MessageStatus.Bounced;
```

Comment:
Codacy found an issue: [Change the visibility of 'MessageStatusBounced' or make it 'const' or 'readonly'.](https://app.codacy.com/manual/stuart_ferguson1/Messaging/pullRequest?prid=5953475)